test/common/apiHelpers.js

```diff
@@ -96,11 +96,11 @@ function getForgingStatus (params, cb) {
 }
 
 function getNodeConstants (cb) {
-	http.get('/api/node/constants', httpCallbackHelper.bind(null, cb));
+	http.get('/LiskHQ/Lisk/1.0.18/node/constants', httpCallbackHelper.bind(null, cb));
```

Comment:
Can we change this base path to a param from upstream?

Reply:
Yes we can do it. I attached validated swagger to the app for this purpose already. Will see how to get rid of the base path from tests.